data-import/src/main/resources/folijet/data-import/features/data-import-delete-logs.feature

```diff
@@ -96,33 +94,30 @@ Feature: Data Import Log deletion tests
 
     * def sourceRecordId = response.entries[0].sourceRecordId
 
-    # retrieve instance hrid from record
+    # Retrieve instance hrid from record
     Given path 'source-storage/records', sourceRecordId
     And headers headersUser
     When method GET
     Then status 200
     And match response.externalIdsHolder.instanceId == '#present'
     * def instanceHrid = response.externalIdsHolder.instanceHrid
 
-    # verify that real instance was created with specific fields in inventory and retrieve instance id
+    # Verify that real instance was created with specific fields in inventory and retrieve instance id
     Given path 'inventory/instances'
     And headers headersUser
     And param query = 'hrid==' + instanceHrid
     When method GET
     Then status 200
     And assert response.totalRecords == 1
     And match response.instances[0].title == '#present'
-    And assert response.instances[0].identifiers[0].identifierTypeId == 'c858e4f2-2b6b-4385-842b-60732ee14abb'
-    And assert response.instances[0].identifiers[0].value == '2020031972'
-    And assert response.instances[0].identifiers[1].identifierTypeId == '439bfbae-75bc-4f74-9fc7-b2a2d47ce3ef'
-    And assert response.instances[0].identifiers[1].value == '(OCoLC)ybp16851676'
-    And assert response.instances[0].notes[0].instanceNoteTypeId == '86b6e817-e1bc-42fb-bab0-70e7547de6c1'
```

Comment:
Hi @AivarIUsupov. Why did you remove asserting for instance note type id and identifier type id?